src/wasi.rs

```diff
@@ -31,65 +28,71 @@ impl Version {
 impl From<&Version> for &'static str {
     fn from(value: &Version) -> Self {
         match value {
-            Version::Snapshot0 => "Snapshot0",
-            Version::Snapshot1 => "Snapshot1",
-            Version::Latest => "Latest",
+            Version::Snapshot0 => "SNAPSHOT0",
+            Version::Snapshot1 => "SNAPSHOT1",
+            Version::Latest => "LATEST",
```

Comment:
Can't comment on it directly because it didn't change, but we may want to make Latest enum value 0 rather than 3, that way it stays stable against future versions of libwasmer where we insert new versions of WASI.

Small detail and being ABI stable this way between versions of libwasmer isn't really a goal we have, but it's nice when things can just work

Reply:
Good catch. I will use `1` instead of `0` because enum in Python starts a `1` as far as I understand them.